docs/source/TechnicalOverview.rst

```diff
@@ -76,18 +76,21 @@ On non-Level 1 platforms, the Land DA System can be run within a container that
 
    Running on Jet, Cheyenne, and NOAA Cloud systems is supported via container. 
 
+.. COMMENT: Has NOAA Cloud support been added natively? If so, add to table above!
```

Comment:
@jkbk2004 Can you answer this? Just need to know if I should add NOAA Cloud to the list of supported Level 1 platforms or if it's still only supported via Singularity.

Reply:
Yes, I will say still "via Singularity". Zach's pw-cloud weather model PR was merged (https://github.com/ufs-community/ufs-weather-model/pull/1667). But Cameron's spack stack weather model pr is not yet (https://github.com/ufs-community/ufs-weather-model/pull/1707).